src/options/generate/generator.py

```diff
@@ -320,3 +347,87 @@ def summarize_error(self, error):
         error_summary = conversation.chat(template_summarize_error.format(error=error))
         return error_summary
 
+    def refine_specification(self):
+        pm = get_random_employee('pm')
+        print(f'{pm.emoji}👋 Hi, I\'m {pm.name}, a PM at Jina AI. Gathering the requirements for our engineers.')
+        self.refine_task(pm)
+        self.refine_test(pm)
+        print(f'''
```

Comment:
Should we ask for confirmation in the end, to confirm description & test?

Reply:
Good idea. We could think about having this in the future.
Let's have a discussion about pros and cons.